contrib/fetch_script_reference.js

```diff
@@ -81,6 +94,7 @@ function is_symbol(str) {
 function fetch_script_category(category) {
     start_task(category);
     var url = get_category_url(category);
+    console.log("category url is : ",url)
```

Comment:
Is this for debug? Can you eliminate it?

Reply:
yes, can remove